src/rendergraph/opengl/CMakeLists.txt

```diff
@@ -0,0 +1,82 @@
+add_library(
+  rendergraph_gl
+  ../common/rendergraph/attributeinit.h
+  ../common/rendergraph/attributeset.h
+  ../common/rendergraph/geometry.h
+  ../common/rendergraph/geometrynode.h
+  ../common/rendergraph/material.h
+  ../common/rendergraph/material/endoftrackmaterial.cpp
+  ../common/rendergraph/material/endoftrackmaterial.h
+  ../common/rendergraph/material/patternmaterial.cpp
+  ../common/rendergraph/material/patternmaterial.h
+  ../common/rendergraph/material/rgbamaterial.cpp
+  ../common/rendergraph/material/rgbamaterial.h
+  ../common/rendergraph/material/rgbmaterial.cpp
+  ../common/rendergraph/material/rgbmaterial.h
+  ../common/rendergraph/material/texturematerial.cpp
+  ../common/rendergraph/material/texturematerial.h
+  ../common/rendergraph/material/unicolormaterial.cpp
+  ../common/rendergraph/material/unicolormaterial.h
+  ../common/rendergraph/materialshader.h
+  ../common/rendergraph/materialtype.h
+  ../common/rendergraph/node.h
+  ../common/rendergraph/opacitynode.h
+  ../common/rendergraph/texture.h
+  ../common/rendergraph/types.h
+  ../common/rendergraph/uniform.h
+  ../common/rendergraph/uniformscache.cpp
+  ../common/rendergraph/uniformscache.h
+  ../common/rendergraph/uniformset.cpp
+  ../common/rendergraph/uniformset.h
+  ../common/types.cpp
```

Comment:
This is repeated multiple times. Can't we define a RENDERGRAPH_COMMON_HEADERS variable in the parent dir and then reference this instead of copy/paste?

Reply:
will do